assets/js/data/cart/tracking.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { getSetting } from '@woocommerce/settings';
+import { Cart, isObject, objectHasProp } from '@woocommerce/types';
+
+let cartPageViewTracked = false;
+let checkoutPageViewTracked = false;
+
+interface StorePageDetails {
+	id: number;
+	title: string;
+	permalink: string;
+}
+
+interface StorePages {
+	checkout: StorePageDetails;
+	cart: StorePageDetails;
+	myaccount: StorePageDetails;
+	privacy: StorePageDetails;
+	shop: StorePageDetails;
+	terms: StorePageDetails;
+}
+export const maybeTrackCheckoutPageView = ( cart: Cart ) => {
+	if ( checkoutPageViewTracked ) {
+		return;
+	}
+	checkoutPageViewTracked = true;
```

Comment:
This mean we retrack on refresh, I don't think that's a major problem

Reply:
Yes the tracking will fire again if the page is reloaded. I would think this is expected.